### ISSUE TITLE
the metrics are quite poor

### ISSUE THREAD

**Issue by z1820785740-cyber** [Aboriginer/HFS-SDE#11]
Could you help clarify my confusion? I performed inference on the FastMRI Knee validation set using the original hfssde weights, but the metrics are quite poor. Do you have any idea what might be causing this?
	SSIM	PSNR	NMSE
1	0.42778095602989197	4.438961505889893	0.6012172102928162
22	0.5986414551734924	5.832489013671875	0.25158894062042236
29	0.652010440826416	19.799137115478516	0.01649625599384308
For the same MRI sample, the results vary significantly across different slices, and none of them reach the performance reported in the paper. Is there anything I can do to improve the results?

<img width="481" height="504" alt="Image" src="https://github.com/user-attachments/assets/90e7536a-668f-405e-9cf1-a9c0701861bd" />

<img width="481" height="504" alt="Image" src="https://github.com/user-attachments/assets/05b90f4d-0272-4017-a6d3-7ab9af42e3d5" />

<img width="481" height="504" alt="Image" src="https://github.com/user-attachments/assets/aa905d2a-9a72-459b-b685-86bfffdf5898" />

**Comment by YuebyYue** [comment]
You can first follow the tuning guidelines in the README to troubleshoot the performance issue, while keeping the other settings unchanged for now, especially the normalization-related ones.

The main part to tune is actually the sampling parameters. In particular:

For `sampling.snr`, try adjusting it gradually with a step size of `0.1`. If needed, you can further do a greedy search with finer steps like `0.01` or `0.02`.

For `sampling.corrector_mse` or `sampling.mse`, there is usually no exact value to target, so it is better to do an exponential search, for example: `0.01`, `0.1`, `1.0`, `10.0`.

I would suggest keeping the rest of the configuration fixed first, and only tuning these two parts to see whether the reconstruction improves.

**Comment by z1820785740-cyber** [comment]
@YuebyYue Thank you for your explanation. I've actually already done that; I just probably haven't found the right values yet. I'll give it another try.

**Comment by z1820785740-cyber** [comment]
@YuebyYue I was wondering if there are reasonable or recommended value ranges for sampling.snr, sampling.corrector_mse, and sampling.mse in the config. I have tried tuning these parameters but haven’t found suitable values yet.

**Comment by YuebyYue** [comment]
@z1820785740-cyber  The best reference values are shown in the repo, but we don't recall whether the dataset patch you're using is aligned with ours. Here are some strategies you could try: First, ensure that all your settings (e.g., normalization, training steps, etc.) are consistent with our method. Second, you can search for appropriate parameters—for example, fix the SNR, then search for the corresponding corrector MSE. To my knowledge, if the artifact is severe, the corrector MSE should be tuned first on an exponential scale, followed by refining the SNR. I hope this is helpful.
